README.md

```diff
@@ -19,16 +19,7 @@ The Data Ingestion Agent (DIA) enables the fast and secure delivery of data into
 
 * Ad Astra Cloud credentials
 * Review the docker [CLI documentation](https://docs.docker.com/engine/reference/commandline/cli/); familiarize yourself with the common commands noted in the [docker section](#docker)
-
-* Docker version 18.02 or greater (Community Edition or any Enterprise Edition)
-  * [Docker download for Windows](https://docs.docker.com/docker-for-windows/install/)
-  * [Docker download for Mac](https://docs.docker.com/v17.12/docker-for-mac/install/)
-  * [Docker download for Linux (Ubuntu)](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
-  * [Docker download for Linux (Debian)](https://docs.docker.com/install/linux/docker-ce/debian/)
-  * [Docker download for Linux (CentOS)](https://docs.docker.com/install/linux/docker-ce/centos/)
-  * [Docker (Enterprise Edition) download for Linux (RHEL)](https://docs.docker.com/install/linux/docker-ee/rhel/)
-    * Installing Docker Community Edition on Red Hat Enterprise Linux (RHEL) will likely require the use of the CentOS guide to be successful
-
+* [Docker version 18.02 or greater](https://docs.docker.com/engine/install/)
```

Comment:
Maybe revert to the original, remove RHEL and add the RHEL help link under CentOS